drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -287,6 +288,8 @@ func (k *k8sSchedOps) validateMountsInPods(
 				if len(pxMounts) > 0 {
 					logrus.Debugf("pod: [%s] %s has PX mount: %v", pod.Namespace, pod.Name, pxMounts)
 					pxMountFound = true
+					// in case there are two pods running with non shared volume, one of them will be in read-only
+					skipHostMountCheck = isMountReadOnly(line)
```

Comment:
@thiguetta how does this prevent from ignoring a read only mount for an actual bug?

So a single (repl 1 deployment) pod mounted a non-shared volume and the mount became read only.

Reply:
I think this answer your question https://jenkins.portworx.co/job/Torpedo/job/tp-master-kops/48/display/redirect

this `if` won't mark pod as validated, so it will keep retrying and skipping in a loop generating a timeout